package.json

```diff
@@ -49,45 +49,46 @@
     "class-validator": "^0.13.2"
   },
   "devDependencies": {
-    "@types/express": "^4.17.6",
-    "@types/express-session": "^1.17.0",
+    "@types/express": "^4.17.14",
+    "@types/express-session": "^1.17.5",
     "@types/form-data": "^2.5.0",
-    "@types/jest": "^26.0.9",
-    "@types/koa": "^2.11.3",
-    "@types/multer": "^1.4.3",
-    "@types/node": "^14.0.11",
-    "@typescript-eslint/eslint-plugin": "^3.8.0",
-    "@typescript-eslint/parser": "^3.8.0",
-    "axios": "^0.21.1",
+    "@types/jest": "^29.2.3",
+    "@types/koa": "^2.13.5",
+    "@types/multer": "^1.4.7",
+    "@types/node": "^16.18.3",
+    "@types/serve-static": "^1.15.0",
+    "@typescript-eslint/eslint-plugin": "^5.44.0",
+    "@typescript-eslint/parser": "^5.44.0",
+    "axios": "^1.2.0",
     "body-parser": "^1.19.0",
     "chakram": "^1.5.0",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.13.2",
     "cors": "^2.8.5",
-    "eslint": "^7.18.0",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-plugin-jest": "^23.20.0",
-    "form-data": "^3.0.0",
+    "eslint": "^8.28.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-jest": "^27.1.6",
+    "form-data": "^4.0.0",
     "handlebars": "^4.7.7",
-    "http-status-codes": "^2.1.4",
-    "husky": "^4.3.8",
-    "jest": "^26.2.2",
+    "http-status-codes": "^2.2.0",
+    "husky": "^8.0.2",
+    "jest": "^29.3.1",
     "kcors": "^2.2.2",
-    "koa-convert": "^1.2.0",
-    "koa-session": "^5.12.3",
-    "koa-views": "^6.2.1",
-    "lint-staged": "^10.2.11",
-    "multer": "^1.4.2",
```

Comment:
Do you know if this version bump resolve this issue [CVE-2022-24434 #882]( https://github.com/typestack/routing-controllers/issues/882)? I believe it should. We are trying to remove all High security vulns from our system and this is our last remaining one.

Reply:
These are the latest versions so I assume it should.

Reply:
Awesome thanks, agreed it should. Let me know if there is anything I could do to help get this pushed across the line!